### PR TITLE
Fix Issues in Edit Command

### PR DESCRIPTION
**Pull request by wujy28**
# Fix Issues in Edit Command
Fixed inconsistencies in attributes for EditCommand and EditCommandParser, as well as fixed a bug
---
# Key changes

## Made attributes utilized in Edit Command consistent
1. Only kept the relevant attributes in EditCommand and EditCommandParser classes and removed the rest

## Fixed bug in PatientBuilder and RecordBuilder
1. Fixed the StackOverflow bug due to the two classes calling on each other's constructors in an endless loop by adding a new RecordBuilder constructor that takes in PatientBuilder as a parameter

## Fixed Checkstyle Issues
1. Fixed all existing checkstyle issues

**NOTE:** There are still 23 bugs to rectify... Guys, please remember to run checkstyle and test cases before pushing if you have not been doing so...

### CODE REVIEW

File: src/test/java/seedu/address/testutil/TypicalPatients.java
Comment (thienmy0) [inline src/test/java/seedu/address/testutil/TypicalPatients.java:32]: I like how you adapted the TypicalPatients class to include our newly added fields!

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java
Comment (thienmy0) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:30]: I like how you changed the import statements since * should be avoided!

**Review by thienmy0 — APPROVED** [review]
I've looked through your changes and couldn't find much issues. Good job! Would be good to wait for the second reviewer before merging

File: src/main/java/seedu/address/model/util/SampleDataUtil.java
Comment (longnguyentan) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:24]: Shall we create constant variables for this (so when we modify the Patient constructor, it would be easier to update this class)?

File: src/test/java/seedu/address/testutil/TypicalPatients.java
Comment (longnguyentan) [reply src/test/java/seedu/address/testutil/TypicalPatients.java:32]: Agree with @thienmy0 on this point :)

**Review by longnguyentan — COMMENTED** [review]
Overall is good :) Thanks for your hard work

**Pull request merged**
